runtime/src/bank.rs

```diff
@@ -2764,6 +2765,22 @@ impl Bank {
         }
     }
 
+    #[allow(dead_code)]
+    fn hash_rewards_into_partitions(
+        stake_rewards: &StakeRewards,
+        parent_block_hash: &Hash,
+        num_partitions: usize,
+    ) -> Vec<StakeRewards> {
+        let hasher = EpochRewardHasher::new(num_partitions, parent_block_hash);
+        let mut result = vec![vec![]; num_partitions];
+
+        for reward in stake_rewards {
+            let partition_index = &hasher.hash_address_to_partition(&reward.stake_pubkey);
+            result[*partition_index].push(reward.clone());
```

Comment:
```suggestion
            let partition_index = hasher.hash_address_to_partition(&reward.stake_pubkey);
            result[partition_index].push(reward.clone());
```

i think? seems like a superfluous reference/dereference by eye anyway.  surprised clippy didn't nag :thinking: